therapeutic/tests/nh_clinical_patient_observation_therapeutic/test_create.py

```diff
@@ -0,0 +1,43 @@
+from openerp.tests.common import SavepointCase
+
+
+class TestCreate(SavepointCase):
+
+    def setUp(self):
+        super(TestCreate, self).setUp()
+        self.test_utils_model = self.env['nh.clinical.test_utils']
+        self.test_utils_model.admit_and_place_patient()
+        self.test_utils_model.copy_instance_variables(self)
+
+        self.therapeutic_observation_model = \
+            self.env['nh.clinical.patient.observation.therapeutic']
+        self.therapeutic_level_model = \
+            self.env['nh.clinical.therapeutic.level']
+
+    def test_sets_patient_therapeutic_level_when_one_exists(self):
+        """
+        If a therapeutic level has ever been set for the patient then a
+        reference to its record should be populated in the `patient_level`
+        field.
```

Comment:
What's the 'patient_level' field? Is this meant to be the therapeutic level shown in the desktop app?

Reply:
The `patient_level` field is the one you were talking about in your previous comment which is not yet shown anywhere. This test will be removed with that field.